api/api-rules/violation_exceptions.list

```diff
@@ -65,6 +65,7 @@ API rule violation: list_type_missing,k8s.io/api/autoscaling/v2beta2,HPAScalingR
 API rule violation: list_type_missing,k8s.io/api/autoscaling/v2beta2,HorizontalPodAutoscalerSpec,Metrics
 API rule violation: list_type_missing,k8s.io/api/autoscaling/v2beta2,HorizontalPodAutoscalerStatus,Conditions
 API rule violation: list_type_missing,k8s.io/api/autoscaling/v2beta2,HorizontalPodAutoscalerStatus,CurrentMetrics
+API rule violation: list_type_missing,k8s.io/api/batch/v1,CronJobStatus,Active
```

Comment:
please add this rather than extend exceptions. cc @apelisse for guidance on list type

Reply:
Syncing with Antoine, will try to fix that.